kaldb/src/test/java/com/slack/kaldb/chunk/ChunkCleanerServiceTest.java

```diff
@@ -117,36 +206,54 @@ public void testDeleteStateDataOn2Chunks() throws IOException {
     final List<LogMessage> messages =
         MessageUtil.makeMessagesWithTimeDifference(1, 11, 1000, startTime);
 
+    int offset = 1;
     for (LogMessage m : messages.subList(0, 9)) {
-      chunkManager.addMessage(m, m.toString().length(), 100);
+      chunkManager.addMessage(m, m.toString().length(), TEST_KAFKA_PARTITION_ID, offset);
+      offset++;
     }
-
     assertThat(chunkManager.getChunkList().size()).isEqualTo(1);
     assertThat(getCount(MESSAGES_RECEIVED_COUNTER, metricsRegistry)).isEqualTo(9);
     assertThat(getCount(MESSAGES_FAILED_COUNTER, metricsRegistry)).isEqualTo(0);
+    testBasicSnapshotMetadata(chunkManager, creationTime);
 
     final ReadWriteChunkImpl<LogMessage> chunk1 = chunkManager.getActiveChunk();
     assertThat(chunk1.isReadOnly()).isFalse();
     assertThat(chunk1.info().getChunkSnapshotTimeEpochMs()).isZero();
 
     for (LogMessage m : messages.subList(9, 11)) {
-      chunkManager.addMessage(m, m.toString().length(), 100);
+      chunkManager.addMessage(m, m.toString().length(), TEST_KAFKA_PARTITION_ID, offset);
+      offset++;
     }
 
+    await().until(() -> getCount(RollOverChunkTask.ROLLOVERS_COMPLETED, metricsRegistry) == 1);
     assertThat(chunkManager.getChunkList().size()).isEqualTo(2);
     assertThat(getCount(MESSAGES_RECEIVED_COUNTER, metricsRegistry)).isEqualTo(11);
     assertThat(getCount(MESSAGES_FAILED_COUNTER, metricsRegistry)).isEqualTo(0);
     assertThat(getCount(RollOverChunkTask.ROLLOVERS_INITIATED, metricsRegistry)).isEqualTo(1);
     assertThat(getCount(RollOverChunkTask.ROLLOVERS_FAILED, metricsRegistry)).isEqualTo(0);
     assertThat(getCount(RollOverChunkTask.ROLLOVERS_COMPLETED, metricsRegistry)).isEqualTo(1);
 
+    List<SnapshotMetadata> snapshots1 = fetchSnapshots(chunkManager);
```

Comment:
Can we use a more descriptive names for these variables? It makes following the test a bit difficult as it's not clear what snapshots are expected to be in this list. (ie, `List<SnapshotMetadata> snapshotsBeforeFoo`)

This applies throughout this class where we have quite a few cases of variables with numbers appended (Ie, `snapshots4`).